src/kava.js

```diff
@@ -675,7 +675,9 @@ class Kava extends BasePlugin {
   }
 
   _onPlaybackRateChanged(): void {
-    this._sendAnalytics(KavaEventModel.SPEED);
+    if (!this.player.playbackRates.length || this.player.playbackRates.includes(this.player.playbackRate)) {
```

Comment:
@semarche-kaltura when would 
this.player.playbackRates.includes(this.player.playbackRate)
be false ?

Reply:
when dual-screen plugin changes playback rate, f.e. it could be 1.05 or 0.873 to synchronise 2 medias (makes the currentTime of 2 medias exact the same)